.idea/dictionaries/common.xml

```diff
@@ -43,7 +43,6 @@
       <w>processmanager</w>
       <w>procman</w>
       <w>proto's</w>
-      <w>protobuf</w>
```

Comment:
Let's add this to `config`'s level dictionary instead of removing here. Especially since it already contains words like `proto's`.

Reply:
This change was caused by a change in `config`. I suppose, IDEA now knows this word by default.